tweetflask.py

```diff
@@ -10,11 +10,16 @@
 app = Flask(__name__)
 
 
-@app.route('/tweets')
-def get():
+def read_file():
     # Opening and reading the db file
     with open('db.json', 'r') as jfile:
         jsondata = json.load(jfile)
+    return jsondata
+
+
+@app.route('/tweets', strict_slashes=False)
```

Comment:
What does `strict_slashes=False` mean?

Reply:
i realized that "https://localhost:5000/tweets/" wasn't going to @app.route('/tweets') and was giving me a 404, and adding the strict_slashes=False fixed it